firetweet/src/main/java/org/getlantern/firetweet/model/Lantern.java

```diff
@@ -39,11 +39,13 @@ public static void start(Context context) {
                 // specify that all of our HTTP traffic should be routed through
                 // our local proxy
 
+                lanternStarted = true;
```

Comment:
I think we should only consider it really initialised when the proxy is started.